integration_tests/pages/bookedtoday/arrivals/matchingRecords/possibleRecordsFound.ts

```diff
@@ -0,0 +1,28 @@
+import Page, { PageElement } from '../../../page'
+
+export default class PossibleRecordsFoundPage extends Page {
+  constructor() {
+    super('Possible existing records have been found')
+  }
+
+  static goTo(id: string): PossibleRecordsFoundPage {
+    cy.visit(`/prisoners/${id}/possible-records-found`)
+    return Page.verifyOnPage(PossibleRecordsFoundPage)
+  }
+
+  perElement = (element): PageElement => cy.get(`.data-qa-per-record-${element}`)
+
+  matchingElement = (element): PageElement => cy.get(`.data-qa-matching-record-${element}`)
+
+  prisonerImage = (): PageElement => cy.get(`[data-qa=prisoner-image]`)
+
+  errorSummary = (): PageElement => cy.get('.govuk-error-summary__list')
+
+  errorMessage = (): PageElement => cy.get('.govuk-error-message')
+
+  searchAgainLink = (): PageElement => cy.get('[data-qa=ammend-search]')
```

Comment:
link is an implementation detail - `searchAgain` should suffice

`...Page.searchAgain().click()`

Reply:
ok